### PR TITLE
Initial FTX decoding

### PR DESCRIPTION
**Pull request by makrsmark**

### CODE REVIEW

File: lib/plugins/Label_H1_FTX.test.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      test('decodes Label H1 Preamble FTX valid', () => {
+        // https://app.airframes.io/messages/3402014738
+        const text = 'FTX/ID23544S,HIFI21,7VZ007B1S276/MR2,/FXFYI .. TAF KSUX 021720Z 0218 0318 20017G28KT P6SM SKC FM022200 22012G18KT P6SM SKC .. PUTS YOUR CXWIND AT 26KT ON RWY 13 .. REDUCES TO 18KT AT 22Z4FEF'
+        const decodeResult = plugin.decode({ text: text });
+        console.log(JSON.stringify(decodeResult, null, 2));
+
+        expect(decodeResult.decoded).toBe(true);
+        expect(decodeResult.decoder.decodeLevel).toBe('partial');
+        expect(decodeResult.raw.flight_number).toBe('HIFI21');
+        expect(decodeResult.raw.mission_number).toBe('7VZ007B1S276');
```
Comment (makrsmark) [inline lib/plugins/Label_H1_FTX.test.ts:32]: specs will fail until #139 is merged

File: lib/plugins/Label_H1_FTX.test.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        plugin = new Label_H1_FTX(decoder);
+    });
+
+    test('matches Label H1 Preamble FTX qualifiers', () => {
+        expect(plugin.decode).toBeDefined();
+        expect(plugin.name).toBe('label-h1-ftx');
+        expect(plugin.qualifiers).toBeDefined();
+        expect(plugin.qualifiers()).toEqual({
+          labels: ['H1'],
+          preambles: ['FTX', '- #MDFTX'],
```
Comment (kevinelliott) [inline lib/plugins/Label_H1_FTX.test.ts:18]: And the formats are both covered? Ok great.

Comment (kevinelliott) [reply lib/plugins/Label_H1_FTX.test.ts:18]: Presumably the same other than the preamble?

**Review by kevinelliott — APPROVED** [review]
Nice. Looks good. We'll have to keep this in mind in terms of how to classify and store in the backend later.

File: lib/plugins/Label_H1_FTX.test.ts (hunk 2 above)
Comment (makrsmark) [reply lib/plugins/Label_H1_FTX.test.ts:18]: correct!

**Pull request merged**